ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -252,6 +208,65 @@ def setup_local_storage(storageclass):
     verify_pvs_created(expected_pvs, storageclass)
 
 
+def create_optional_operators_catalogsource_non_ga(force=False):
+    """
+    Creating optional operators CatalogSource and ImageContentSourcePolicy
+    for non-ga OCP.
+
+    Args:
+        force (bool): enable/disable lso catalog setup
+
+    """
+    ocp_version = version.get_semantic_ocp_version_from_config()
+    ocp_ga_version = get_ocp_ga_version(ocp_version)
+    if ocp_ga_version and not force:
+        return
```

Comment:
But you no longer need to fetch version and skip the function based on that here again, right?

Reply:
Correct.. But I need to add condition here too
https://github.com/red-hat-storage/ocs-ci/blob/0d0522b469e0980097fbaf8766fec66389003676/ocs_ci/ocs/ui/deployment_ui.py#L422

@mbukatov What do you think? remove the condition in `create_optional_operators_catalogsource_non_ga`?